turbo/jsonrpc/zkevm_api.go

```diff
@@ -721,94 +717,80 @@ func (api *ZkEvmAPIImpl) fullTxBlockData(ctx context.Context, tx kv.Tx, hermezDb
-	if batchNum == 0 && prevSequence.BatchNo == 0 {
-		return &common.Hash{}, nil
-	}
-
-	if prevSequence == nil || batchSequence == nil {
-		var missing string
-		if prevSequence == nil && batchSequence == nil {
-			missing = "previous and current batch sequences"
-		} else if prevSequence == nil {
-			missing = "previous batch sequence"
-		} else {
-			missing = "current batch sequence"
-		}
-		return nil, fmt.Errorf("failed to get %s for batch %d", missing, batchNum)
-	}
-
-	// get batch range for sequence
-	prevSequenceBatch, currentSequenceBatch := prevSequence.BatchNo, batchSequence.BatchNo
-	// get call data for tx
-	l1Transaction, _, err := api.l1Syncer.GetTransaction(batchSequence.L1TxHash)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get transaction data for tx %s: %w", batchSequence.L1TxHash, err)
-	}
-	sequenceBatchesCalldata := l1Transaction.GetData()
-	if len(sequenceBatchesCalldata) < 10 {
-		return nil, fmt.Errorf("calldata for tx %s is too short", batchSequence.L1TxHash)
-	}
-
-	currentBatchForkId, err := db.GetForkId(currentSequenceBatch)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get fork id for batch %d: %w", currentSequenceBatch, err)
-	}
-
-	prevSequenceAccinputHash, err := api.GetccInputHash(ctx, currentBatchForkId, prevSequenceBatch)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get old acc input hash for batch %d: %w", prevSequenceBatch, err)
-	}
-
-	decodedSequenceInterface, err := syncer.DecodeSequenceBatchesCalldata(sequenceBatchesCalldata)
-	if err != nil {
-		return nil, fmt.Errorf("failed to decode calldata for tx %s: %w", batchSequence.L1TxHash, err)
-	}
-
-	accInputHashCalcFn, totalSequenceBatches, err := syncer.GetAccInputDataCalcFunction(batchSequence.L1InfoRoot, decodedSequenceInterface)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get accInputHash calculation func: %w", err)
-	}
-
-	if totalSequenceBatches == 0 || batchNum-prevSequenceBatch > uint64(totalSequenceBatches) {
-		return nil, fmt.Errorf("batch %d is out of range of sequence calldata", batchNum)
-	}
-
-	// calculate acc input hash
-	accInputHash = &prevSequenceAccinputHash
-	for i := 0; i < int(batchNum-prevSequenceBatch); i++ {
-		accInputHash = accInputHashCalcFn(prevSequenceAccinputHash, i)
-		prevSequenceAccinputHash = *accInputHash
-	}
-
-	return
-}
-
-func (api *ZkEvmAPIImpl) GetccInputHash(ctx context.Context, currentBatchForkId, lastSequenceBatchNumber uint64) (accInputHash common.Hash, err error) {
-	if currentBatchForkId < uint64(chain.ForkID8Elderberry) {
-		accInputHash, err = api.l1Syncer.GetPreElderberryAccInputHash(ctx, &api.config.AddressRollup, lastSequenceBatchNumber)
-	} else {
-		accInputHash, err = api.l1Syncer.GetElderberryAccInputHash(ctx, &api.config.AddressRollup, api.config.L1RollupId, lastSequenceBatchNumber)
-	}
-
-	if err != nil {
-		err = fmt.Errorf("failed to get accInputHash batch %d: %w", lastSequenceBatchNumber, err)
-	}
-
-	return
-}
+//type SequenceReader interface {
```

Comment:
we can delete instead of commenting, and if we need we can check for  commits ?

or better to keep commented code on code base ?

Reply:
For sure - not worth review for stuff like this yet still massively wip